portfolio/src/main/webapp/script.js

```diff
@@ -26,3 +26,12 @@ function addRandomGreeting() {
   const greetingContainer = document.getElementById('greeting-container');
   greetingContainer.innerText = greeting;
 }
+
+/**
+ * Displays a greeting to the front page.
+ */
+async function getGreeting(){
+    const response = await fetch('/data');
```

Comment:
Nice! I might ask you to give a brief lesson on async and await in our meeting :cat2: